### ISSUE TITLE
Flush-ContainerHelperCache Total reclaimed space

### ISSUE THREAD

**Issue by mynavblog** [microsoft/navcontainerhelper#1613]
**Describe the issue**
The Total reclaimed space shows 0B even if working fine and delete the data correctly

**Scripts used to create container and cause the issue**
```
Flush-ContainerHelperCache -keepDays 1
```

**Full output of scripts**
```
BcContainerHelper version 1.0.19
WARNING: The names of some imported commands from the module 'BcContainerHelper' include unapproved verbs that might make them less discoverable. To find the commands wit
h unapproved verbs, run the Import-Module command again with the Verbose parameter. For a list of approved verbs, type Get-Verb.
Flush-ContainerHelperCache -keepDays 1
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20449\platform
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20449\us
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20486\platform
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20486\us
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20494\platform
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20494\us
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20503\platform
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20503\us
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20601\platform
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20601\us
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20652\platform
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20652\us
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20697\platform
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20697\us
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20739\platform
Removing C:\bcartifacts.cache\sandbox\17.2.19367.20739\us
Total reclaimed space: 0B
...
```

**Screenshots**
![image](https://user-images.githubusercontent.com/37371997/104442405-4c171000-5595-11eb-85ea-98214b591118.png)

**Additional context**
I do not think it is a big issue but confusing a little when you see that data was deleted and over 14GB was reclaimed.

**Comment by freddydk** [comment]
The last line actually comes from Docker System Prune (and is not representing the full flush).
I will pipe this to null and avoid the line.

**Comment by freddydk** [comment]
Shipped in BcContainerHelper 2.0.0
